wca-integration/src/main/java/eu/itesla_project/wca/WCAImpl.java

```diff
@@ -318,11 +318,15 @@ private static void copyRequired(Path workingDir) throws IOException {
 
                         // write base state
                         new AmplNetworkWriter(network, dataSource, 0, 0, false, mapper, CLUSTERS_AMPL_EXPORT_CONFIG).write();
+                        if (config.isExportStates())
+                            new WCAStateExporter(network, workingDir, 0, 0).export();
 
                         // write post contingency state
                         network.getStateManager().setWorkingState(contingencyStateId);
                         AmplUtil.fillMapper(mapper, network); // because action can create a new bus
                         new AmplNetworkWriter(network, dataSource, contingencyNum, 0, true, mapper, CLUSTERS_AMPL_EXPORT_CONFIG).write();
+                        if (config.isExportStates())
+                            new WCAStateExporter(network, workingDir, contingencyNum, 0).export();
```

Comment:
Add braces to respect java coding guideline even if there is only one statement.